cobridge_base/include/common.hpp

```diff
@@ -31,6 +31,8 @@ using ClientChannelId = uint32_t;
 using SubscriptionId = uint32_t;
 using ServiceId = uint32_t;
 
+// coBridge Websocket Protocol
+// https://coscene0.feishu.cn/wiki/T8Enwstxni9Cq7k760FcFiU7n7f
```

Comment:
Since coBridge is open source project, if we want to reference any docs link, it is better to add it in github.